.github/workflows/ci.yml

```diff
@@ -301,6 +303,9 @@ jobs:
           restore-keys: ${{ github.job }}-ccache-
 
       - name: CI script
+        env:
+          # Using the "Release" build type to avoid oversized executables to pass to the following job.
```

Comment:
What is "oversized" here, is there some bandwidth constraint? I'm guessing no debug symbols also makes things less useful for potential debugging in the CI.

Reply:
This change has been dropped.